perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/AuthzResolverMethod.java

```diff
@@ -371,7 +377,8 @@ public User call(ApiCaller ac, Deserializer parms) throws PerunException {
 	 * including user identification, authorization and metadata. Each call of this method refresh the
 	 * session including authorization data.
 	 *
-	 * @return PerunPrincipal PerunPrincipal
+	 * @deprecated
```

Comment:
This method is marked as deprecated in core api layer, but it shouldn't. It's part of production edu gain hack branch and is currently used. Just remove ``@deprecated``. It will get in sync with core api in some time. Thanks.